docs/spark-connector/spark-catalog-iceberg.md

```diff
@@ -111,6 +111,7 @@ Gravitino spark connector will transform below property names which are defined
 | `io-impl`                       | `io-impl`                             | The io implementation for `FileIO` in Iceberg.                                                                                                                                                                      | 0.6.0         |
 | `s3-endpoint`                   | `s3.endpoint`                         | An alternative endpoint of the S3 service, This could be used for S3FileIO with any s3-compatible object storage service that has a different endpoint, or access a private S3 endpoint in a virtual private cloud. | 0.6.0         | 
 | `s3-region`                     | `client.region`                       | The region of the S3 service, like `us-west-2`.                                                                                                                                                                     | 0.6.0         |
+| `oss-endpoint`                  | `oss.endpoint`                        | The endpoint of Aliyun OSS service.                                                                                                                                                                                 | 0.7.0         |
```

Comment:
should you also add access key and secret access key here in Spark doc?

Reply:
No, because Gravitino doesn't export access key and secret key to users for security. User should set the security key explictly.